### ISSUE TITLE
The pages need to have .html ending!

### ISSUE THREAD

**Issue by KaariL** [KaariL/jacqs#1]
we need to make sure our web browser knows to read the page in the html language, so we gotta end the pages with that file type.

**Comment by jasmithers** [comment]
of course. I thought Cuz I created a page that it would show up as html. I know they all have to be that.
